README.md

```diff
@@ -1 +1,23 @@
-# TBA
+# SimpleIndexer
+
+
+## Usage
+
+#### via Docker image (recommended)
+
+```python
+from jina import Flow
+	
+f = Flow().add(uses='jinahub+docker://SimpleIndexer')
+```
+
+#### via source code
+
+```python
+from jina import Flow
+	
+f = Flow().add(uses='jinahub://SimpleIndexer')
+```
+
+- To override `__init__` args & kwargs, use `.add(..., uses_with: {'key': 'value'})`
+- To override class metas, use `.add(..., uses_metas: {'key': 'value})`
```

Comment:
unnecessary, delete it
```suggestion
```

Reply:
e.g. SimpleIndexer will store the documents in a folder that is specified by `workspace` (which can be defined via ...). Inside this `workspace` there are two files: `head.bin`, `body.bin` which corresponds to the file structure of `DocumentArrayMemmap`

Reply:
two designs of SimpleIndexer
1. it leverages the `DAM` in the core
2. it leverages the `.match` in the core


To leverage the full flexibility of `.match`, one can specify `match_args` in `SimpleIndexer`, e.g. 

```python

f = Flow().add(uses=SimpleIndexer, uses_with={'match_args': {'lhs_traversal_path': ['c', 'm']}})

```